<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewPageWidget</class>
 <widget class="QWidget" name="ReviewPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>1</y>
     <width>1022</width>
     <height>1022</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget
{
background:#1F2327;
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="idlePage">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="pic">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>420</y>
       <width>136</width>
       <height>134</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>136</width>
       <height>134</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>136</width>
       <height>134</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
image:url(&quot;:/FunctionalWidget/FunctionalWidget/Review/Resource/noperson.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>640</y>
       <width>376</width>
       <height>80</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">QLabel
{
font: 28px &quot;Microsoft YaHei&quot;;
color:rgba(180,196,217,1);
}</string>
        </property>
        <property name="text">
         <string>在“</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="person">
        <property name="styleSheet">
         <string notr="true">QLabel
{
image:url(&quot;:/FunctionalWidget/FunctionalWidget/Review/Resource/patient.png&quot;);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">QLabel
{
font: 28px &quot;Microsoft YaHei&quot;;
color:rgba(180,196,217,1);
}</string>
        </property>
        <property name="text">
         <string>”中新建或查看患者</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="reivewPage"/>
   <widget class="QWidget" name="exposePage">
    <widget class="QWidget" name="gridWidget" native="true">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>450</y>
       <width>1021</width>
       <height>197</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="pic_2">
          <property name="minimumSize">
           <size>
            <width>136</width>
            <height>134</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>136</width>
            <height>134</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel
{
image:url(&quot;:/FunctionalWidget/FunctionalWidget/Review/Resource/Expose.png&quot;);
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <item alignment="Qt::AlignRight">
         <widget class="QLabel" name="name">
          <property name="styleSheet">
           <string notr="true">QLabel
{
font: 28px &quot;Microsoft YaHei&quot;;
color:rgba(180,196,217,1);
}</string>
          </property>
          <property name="text">
           <string>张思远</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">QLabel
{
font: 28px &quot;Microsoft YaHei&quot;;
color:rgba(180,196,217,1);
}</string>
          </property>
          <property name="text">
           <string>正在拍片</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../CArmWorkStation.qrc"/>
 </resources>
 <connections/>
</ui>
